.github/workflows/osx_testing.yml

```diff
@@ -0,0 +1,185 @@
+name: testing
```

Comment:
Nit: I would prefer to keep the workflow name the same as the workflow file name (`osx_testing` in the case).

Reply:
Thanks, fixed.